conda/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+$PYTHON -m pip install .
+
+CMAT="${PREFIX}/share/${PKG_NAME}-${PKG_VERSION}"
+mkdir -p ${PREFIX}/bin ${CMAT}
+
+chmod 775 bin/cmat/*
+cp bin/cmat/* ${PREFIX}/bin
+
+mv bin/ mappings/ pipelines/ ${CMAT}
```

Comment:
Regarding how we handle ``bin/`` in the build, we are assuming that it will always be done within a virtual environment, right? 

Couldn't this be a big problem if someone was to install it directly in their main env. where everything in ``bin`` may be moved to the CMAT directory?

Reply:
This doesn't get run by users, only by conda build (which creates a separate env to manage)... we can mention in the documentation that people shouldn't run it themselves.